curriculum/challenges/english/01-responsive-web-design/css-piano/part-006.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 612e78af05201622d4bab8aa
+title: Part 6
+challengeType: 0
+dashedName: part-6
+---
+
+# --description--
+
+Time to start working on the piano. Within your `body` element, create a `div` element.
```

Comment:
**Note**: I would combine the next 4 lessons with this one - Campers have done a lot of this, and should be able to do this very easily, by now.

That is, IMO, there are too many lessons Campers have done too many times.